### ISSUE TITLE
Hexagon dimensions picture

### ISSUE THREAD

**Issue by finnp** [terinjokes/StickerConstructorSpec#8]
I am always looking for something like this, so I just created it for people who need it:

![hexagon](https://cloud.githubusercontent.com/assets/841788/4592665/658a8238-507e-11e4-9430-9ff7268a73fe.png)

Also: Can we rename this to HexagonStickerSpec? There is a square sticker mentioned in the spec, but I think this is out of the scope of this project ;).

**Comment by rstacruz** [comment]
Can we get @finnp's image in there somewhere? It took me forever to find it :(

**Comment by terinjokes** [comment]
@finnp I added your image to the README, I'd love to get a square still to close out this ticket.

**Comment by mairin** [comment]
Hi, I'm a free software using designer and made both a template for the hexagonal sticker and for the square sticker as detailed in the spec. I followed Sticker Mule's template style, and allowed for a 0.125" bleed. Since I think it makes sense, I also included a border in the sticker design that is 0.1" in the final trimmed design (I went through a bunch of the pre-existing stickers and determined this is about the average width of the borders on the stickers that use borders.)

They are made in Inkscape, for Inkscape. Each includes 3 layers, "Template Text" with the instructions, "Artwork" for the sticker artworm, "Trim Lines" for the printer's trim / bleed / safe area lines, and "Preview without bleeds" which is a layer you can turn on to mask out the bleed so you can see how the final sticker will look.

Github won't accept SVG uploads so I'm going to link to the templates I made:

https://duffy.fedorapeople.org/sticker-templates/hexagonal-sticker_template.svg

https://duffy.fedorapeople.org/sticker-templates/square-sticker_template.svg

**Comment by terinjokes** [comment]
@mairin :+1: thanks. Any objections of forking these and including them in the repo?

**Comment by mairin** [comment]
Not at all, will do a PR now, just wanted to make sure they would cool with you.

**Comment by mairin** [comment]
https://github.com/terinjokes/StickerConstructorSpec/pull/17
